src/storage/data_table.cpp

```diff
@@ -76,7 +76,7 @@ DataTable::SlotIterator &DataTable::SlotIterator::operator++() {
 DataTable::SlotIterator DataTable::end() const {  // NOLINT for STL name compability
   // TODO(Lin): We need to temporarily comment out this latch for the concurrent TPCH experiments. Should be replaced
   //  with a real solution
-  common::SpinLatch::ScopedSpinLatch guard(&blocks_latch_);
+  // common::SpinLatch::ScopedSpinLatch guard(&blocks_latch_);
```

Comment:
I think this shouldn't be committed into master?

Reply:
Fixed.